pkg/event/event.go

```diff
@@ -7,12 +7,14 @@ import (
 	"errors"
 	"time"
 
+	"github.com/inngest/inngest/pkg/consts"
 	"github.com/oklog/ulid/v2"
 )
 
 const (
 	EventReceivedName = "event/event.received"
 	FnFailedName      = "inngest/function.failed"
+	FnCompletedName   = "inngest/function.completed"
```

Comment:
This should probably be named `inngest/function.finished` to be inkeeping with other terminology.

This also doesn't fit nicely with `inngest/function.failed`. The downside is that we'd have to create two pauses when waiting for invocation results if we had the two events being sent. My lean is that we ditch `inngest/function.failed` and consolidate everything into the `inngest/function.finished` event, though this would likely break some user flows if they have manually targetted that event.

Reply:
100% on `function.finished`.  Ditching `inngest/function.failed` is backcompat difficult.  A migration plan:

1. We start sending one event now: `inngest/function.completed` with a status of success/error/cancel, etc. while keeping `inngest/function.failed.
2. We update the SDK's function handlers to listen to these events with an expression
3. Eventually we deprecate `inngest/function.failed`

That said, failure handling is _so common_ that I don't hate special casing it.